basicrta/gibbs.py

```diff
@@ -52,6 +52,15 @@ def run(self, run_resids=None, g=100):
         with open(self.contacts, 'r+b') as f:
             contacts = pickle.load(f)
 
+        # Check if this is a combined contact file
+        metadata = contacts.dtype.metadata
+        is_combined = metadata and 'n_trajectories' in metadata and metadata['n_trajectories'] > 1
+        if is_combined:
+            print(f"WARNING: Using combined contact file with {metadata['n_trajectories']} trajectories.")
+            print("WARNING: Kinetic clustering is not yet supported for combined contacts.")
+            print("WARNING: The Gibbs sampler will pool all residence times together.")
+            print("WARNING: Trajectory source information is available but not used in kinetic clustering.")
```

Comment:
This function does NOT perform kinetic clustering (also know as "kinetic mapping"). The kinetic mapping functionality lives in kinetics.KineticMap. You can remove line 62.